articles/azure-monitor/app/opencensus-python.md

```diff
@@ -46,160 +45,113 @@ First you have to create an Application Insights resource which will generate an
     ```
 
-4. While this is helpful for demonstration purposes, ultimately we want to emit the SpanData in a way that it can be picked up by our **local forwarder service** and sent on to Application Insights. Modify your code from the previous step to the following:
+4. While this is helpful for demonstration purposes, ultimately we want to emit the SpanData to Application Insights. Modify your code from the previous step to the following:
 
     ```python
+    from opencensus.ext.azure.trace_exporter import AzureExporter
+    from opencensus.trace.samplers import ProbabilitySampler
     from opencensus.trace.tracer import Tracer
-    from opencensus.trace import config_integration
-    from opencensus.ext.ocagent.trace_exporter import TraceExporter
-    from opencensus.trace import tracer as tracer_module
-
-    import os
-
-    def main():
-        while True:
-            valuePrompt()
+    
+    # TODO: replace the all-zero GUID with your instrumentation key.
+    tracer = Tracer(
+        exporter=AzureExporter(
+            instrumentation_key='00000000-0000-0000-0000-000000000000',
+        ),
+        sampler=ProbabilitySampler(1.0),
+    )
 
     def valuePrompt():
-        export_LocalForwarder = TraceExporter(
-        service_name=os.getenv('SERVICE_NAME', 'python-service'),
-        endpoint=os.getenv('OCAGENT_TRACE_EXPORTER_ENDPOINT'))
-
-        tracer = Tracer(exporter=export_LocalForwarder)
         with tracer.span(name="test") as span:
             line = input("Enter a value: ")
             print(line)
 
+    def main():
+        while True:
+            valuePrompt()
+
     if __name__ == "__main__":
         main()
-
     ```
-
-5. If you save and try running the above module, you may receive a `ModuleNotFoundError` for `grpc`. If this occurs run the following to install the [grpcio package](https://pypi.org/project/grpcio/) with:
-
-    ```console
-    python -m pip install grpcio
-    ```
-
-6. Now when you run the Python script from above you should still be prompted to enter values, but now only the value is being printed in the shell.
-
-7. To confirm that the **local forwarder** is picking up the traces check the `LocalForwarder.config` file. If you followed the steps in the [prerequisite](https://docs.microsoft.com/azure/application-insights/local-forwarder), it will be located in `C:\LF-WindowsServiceHost`.
-
-    In the image below of the log file, you can see that prior to running the second script where we added an exporter `OpenCensus input BatchesReceived` was 0. Once we started running the updated script `BatchesReceived` incremented equal to the number of values we entered:
-    
-    ![New App Insights resource form](./media/opencensus-python/0004-batches-received.png)
+5. Now when you run the Python script from above you should still be prompted to enter values, but now only the value is being printed in the shell.
```

Comment:
A little awkward phrasing with two "now"s. Maybe: "When you run the Python script from above, you should still be prompted to enter values. However the value is only being printed in the shell.